server/src/models/user.model.ts

```diff
@@ -1,23 +1,19 @@
 import { Schema, model, models, Types } from 'mongoose';
+import { UserType } from '../types/user.types';
 
-interface IUser {
-  fullName: string;
-  email: string;
-  password: string;
-  accessType: string;
-  groupIds: Types.Array<Types.ObjectId>;
-  setlistIds: Types.Array<Types.ObjectId>;
-}
+const userSchema = new Schema<UserType>(
+  {
+    fullName: { type: String, required: true },
+    email: { type: String, required: true, unique: true },
+    password: { type: String, required: true },
+    accessType: { type: String, enum: ['admin', 'normal'], default: 'normal' },
```

Comment:
wasn't there more than just these 2 access types? like t3ch, band/cw, tc, member, etc. I'm not sure what are the proper names but can u double check with cxo?